.github/scripts/distfiles.sh

```diff
@@ -0,0 +1,65 @@
+#!/usr/bin/env bash
+# Copyright (C) Viktor Szakats
+#
+# SPDX-License-Identifier: curl
+
+# Compare git repo files with tarball files and report a mismatch
+# after excluding exceptions.
+
+set -eu
+
+tarfiles="$(mktemp)"
+gitfiles="$(mktemp)"
+
+taronly="/
+^
+Makefile\.in
+^aclocal.m4
+^compile
+^configure
+^config\..+
+^depcomp
+^install-sh
+^ltmain\.sh
+^missing
+^docs/libcurl/libcurl-symbols\.md
+^docs/RELEASE-TOOLS\.md
+^docs/tarball-commit\.txt
+^lib/curl_config\.h\.in
+^m4/libtool\.m4
+^m4/lt.+\.m4
+^src/(tool_ca_embed|tool_hugehelp)\.c"
+
+gitonly="\.git.+
+^\..+
+^appveyor\..+
+^buildconf
+^renovate\.json
+^REUSE\.toml
+^(GIT-INFO|README|SECURITY)\.md
+^LICENSES/.+
+^docs/examples/adddocsref\.pl
+^docs/THANKS-filter
+^lib/optiontable\.pl
```

Comment:
Since `optiontable.pl` is used in a lib/Makfile.am target and test 1276, I think we should rather include it in the dist.

Reply:
Updated!